css/style.css

```diff
@@ -1,19 +1,200 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
+
+
 body {
+
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+/* Header */
+
+.header {
+    
+    Height:96px;
+    padding: 1rem;
+    display: flex;
+    justify-content:space-around;
+    align-items: center;
+    background-color: #C4C4C41A;
+}
+
+.header-logo {
+height: 70px;
+width: 70px;
+}
+
+ /* navigation */
+.nav-list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+}
+.nav-item{
+padding: 1rem 1rem;
+margin: 0.3rem;
+
+}
+
+.nav-link{
+    font-size: 20px;
+    text-decoration: none;
+    color: #8e929a;
+}
+.nav-link:hover{
+    color:#d06a45;
+}
+
+.navigation li:first-child a {
+ color: rgb(48, 48, 48);
+}
+
+
+/* hero */
+
+.hero {
+    background-image: url("../img/first-background.jpg");
+    background-repeat:no-repeat ;
+    height:90vh;
+    background-size: cover;
+    color: white;
+    text-align: center;
+    border: 1px solid lightgray;
+}
+.hero h1{
+    padding-top: 18.5rem;
```

Comment:
I really like your job. The design is very similar to the sketch.
But to centralise the elements you use - padding-top. In your screen it should looks good but in mine for example in hero section the title and button go beyond the hero section.
I can suggest to use a feature:
wrap all elements in some <div> : 

        <section class="hero" >
            <div class="hero-container">
                <h1>Introducing Karma</h1>
                <p>Bring WiFi with you,everywhere you go.</p>
                <button class="learn-btn">Learn More</button>
            </div>
        </section>

And centralize it by using transform:

add  in hero:
.hero {
    position: relative;
}

.hero-container {
    position: absolute;
    top: 20%;
    left: 50%;
    transform: translate(-50%, -50%);
    text-align: center;
}

Or you can use Flex for that.

Reply:
In your order-form you can also correct the position of some fields. I used flexbox for this.

Reply:
but overall - good job